pkg/autoupdate/handler.go

```diff
@@ -1,52 +1,115 @@
-// +build !windows
-
 package autoupdate
 
 import (
 	"os"
 	"path/filepath"
+	"strconv"
+	"time"
 
 	"github.com/go-kit/kit/log/level"
 	"github.com/kolide/kit/fs"
 	"github.com/kolide/updater/tuf"
 )
 
-// handler is called by the tuf package when tuf detects a change with
-// the remote metadata.
+// handler is called by the tuf package in two cases. First, and
+// confusingly, it's used as an error reporting channel for any kind
+// of issue. In this case, it's called with an err set.
+//
+// Second, it's called when tuf detects a change with the remote metadata.
 // The handler method will do the following:
 // 1) untar the staged staged file,
-// 2) replace the existing binary,
+// 2) into the updated directory
```

Comment:
is this untar-ing the staged file into `<binaryPath>-updates/`? I think it's being put into a `<binaryPath>-updates/<somedatehere>/` directory right?